content/en/docs/ops/common-problems/security-issues/index.md

```diff
@@ -84,17 +84,11 @@ matching requests should flow through. If all requests continue to be denied, yo
 
 1. Make sure there is no typo in your policy YAML file.
 
-1. Avoid enabling authorization for Istio Control Planes Components, including Mixer, Pilot, Ingress. Istio authorization policy is designed for authorizing access to services in Istio Mesh. Enabling it for Istio Control Planes Components may cause unexpected behavior.
+1. Avoid enabling authorization for istiod. Istio authorization policy is designed for authorizing access to workloads in Istio Mesh. Enabling it for istiod may cause unexpected behavior.
 
-1. Make sure that your `ServiceRoleBinding` and referred `ServiceRole` objects are in the same namespace (by checking "metadata"/”namespace” line).
+1. Make sure that your authorization policies are in the right same namespace (by checking "metadata"/”namespace” line).
 
-1. Make sure that your service role and service role binding policies don't use any HTTP only fields
-for TCP services. Otherwise, Istio ignores the policies as if they didn't exist.
-
-1. In Kubernetes environment, make sure all services in a `ServiceRole` object are in the same namespace as the
-`ServiceRole` itself. For example, if a service in a `ServiceRole` object is `a.default.svc.cluster.local`, the `ServiceRole` must be in the
-`default` namespace (`metadata/namespace` line should be `default`). For non-Kubernetes environments, all `ServiceRoles` and `ServiceRoleBindings`
-for a mesh should be in the same namespace.
+1. Make sure that your authorization policies don't use any HTTP only fields for TCP traffic. Otherwise, Istio ignores the policies as if they didn't exist.
 
```

Comment:
didn't exist =>don't exist. 

It that true for deny policies?

Reply:
Yeah, it's slightly different here. Updated and added another bullet point for deny policy.

Reply:
Looks good. Thanks.